modules/olm-catalogsource-image-template.adoc

```diff
@@ -12,20 +12,20 @@ endif::[]
 [id="olm-catalogsource-image-template_{context}"]
 = Image template for custom catalog sources
 
-Operator compatibility with the underlying cluster can be expressed by a catalog source in various ways. One way, which is used for the default Red Hat-provided catalog sources, is to identify image tags for index images that are specifically created for a particular platform release, for example {product-title} 4.13.
+Operator compatibility with the underlying cluster can be expressed by a catalog source in various ways. One way, which is used for the default Red Hat-provided catalog sources, is to identify image tags for index images that are specifically created for a particular platform release, for example {product-title} 4.14.
 
-During a cluster upgrade, the index image tag for the default Red Hat-provided catalog sources are updated automatically by the Cluster Version Operator (CVO) so that Operator Lifecycle Manager (OLM) pulls the updated version of the catalog. For example during an upgrade from {product-title} 4.12 to 4.13, the `spec.image` field in the `CatalogSource` object for the `redhat-operators` catalog is updated from:
+During a cluster upgrade, the index image tag for the default Red Hat-provided catalog sources are updated automatically by the Cluster Version Operator (CVO) so that Operator Lifecycle Manager (OLM) pulls the updated version of the catalog. For example during an upgrade from {product-title} 4.13 to 4.14, the `spec.image` field in the `CatalogSource` object for the `redhat-operators` catalog is updated from:
```

Comment:
There are several instances of `4.14` in this module. I presume you bump these each release. Make sense to replace with our version variable?

Reply:
Actually @sheriff-rh has been taking care of the 4.13->4.14 version bumps (I only did it in this file because it didn't make sense to bump the kube versions without bumping the OCP versions too). I believe has a PR open that does change some of these hard-coded versions to use the version attribute. (His PR and mine might even conflict, if it hasn't been merged yet. But that's okay)

However, in this case I don't think it will work because we don't have an attribute for the previous version (4.13). So we'd want to keep them hard-coded so we remember to come back and update both versions each release. 

Thanks for asking the question though!

Reply:
Correct, @bergerhoffer mine is still in progress. I need to ask for more reviews prior to merge. Thanks for thinking of me! I'll be sure to rebase and correct as needed before merging!

Reply:
Done and done :) Go team!